converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java

```diff
@@ -32,6 +34,7 @@
 	private boolean anded;
 	private Map<TemplateId, AtomicInteger> nodeCount;
 	private Comparable<?> lastAppraised;
+	private static final String DATE_FORMAT = "yyyyMMdd";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "DATE_FORMAT" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)